recipes/suffix-tree/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "suffix-tree" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/suffix_tree-{{ version }}.tar.gz
+  sha256: e5a927d205956c0161ed4fc90959e55326f67bccea1291fa476b408f347d92ef
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
```

Comment:
Please have a look at the `noarch: python` documentation in terms of `{{ python_min }}`:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python